alacritty/src/input/keyboard.rs

```diff
@@ -644,3 +643,11 @@ impl From<ModifiersState> for SequenceModifiers {
         modifiers
     }
 }
+
+/// Check whether the text represents a C0 or C1 codepoint.
+fn is_c0c1_codepoint(text: &str) -> bool {
```

Comment:
Might want to change the name because of `0x7f`? Maybe just `is_control_character` with a comment which specific control characters this function detects?